docs/src/pages/demos/autocomplete/Autosuggest.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import match from 'autosuggest-highlight/match';
+import parse from 'autosuggest-highlight/parse';
+import MenuItem from '@material-ui/core/MenuItem';
+import Paper from '@material-ui/core/Paper';
+import { Theme } from '@material-ui/core/styles';
+// import TextField from '@material-ui/core/TextField';
+import { makeStyles } from '@material-ui/styles';
```

Comment:
```suggestion
import { makeStyles } from '@material-ui/core/styles';
```

Reply:
@merceyz, can you please explain this suggestion? My understanding is that `@material-ui/core/styles` is the old styling system, whereas in the .next branch we should be using the new typing system.

Does this suggestion make the autocomplete demo page come up?

Reply:
`@material-ui/core/styles` exports the `@material-ui/styles` functions but with a default theme.

Reply:
Thanks for the explanation @joshwooding. I have committed @merceyz's suggestion to my fork. Still not able to bring up the example :-(. Pretty much the same example is working in my [other repo](https://github.com/nareshbhatia/stock-charts) which was create with create-react-app. Would be great help to solve this issue.

Reply:
I’m on my phone at the moment, I can take a look in a bit. ( If someone doesn’t beat me to it :P )